### ISSUE TITLE
Invalid + Inconsistent Type Name Escaping

### ISSUE THREAD

**Issue by olirice** [eulerto/wal2json#224]
# Description

- Format version 2 type escaping behavior results in invalid json
- Type name escaping behavior is inconsistent for type names when compared to column/table names
- Type name escaping behavior is inconsistent between format version 1 & 2

### Setup
```sql
create type "MOOD" as enum ('happy', 'sad');

create table public."Test" ( 
	id bigint primary key,
	mood "MOOD"
);

select * from pg_create_logical_replication_slot('test_case', 'wal2json');

insert into public."Test"(id, mood) values (1, 'happy');
```

### Format Version 1 Behavior
```sql
select
	data
from
	 pg_logical_slot_peek_changes(
		'test_case', null, null,
		'include-transaction', 'false',
		'format-version', '1'
	) x
```

response
```json
{
  "change": [
    {
      "kind": "insert",
      "table": "Test", -- correctly formatted
      "schema": "public",
      "columnnames": [
        "id",
        "mood"
      ],
      "columntypes": [
        "bigint",
        "\"MOOD\"" -- unexpected escaping difference
      ],
      "columnvalues": [
        1,
        "happy"
      ]
    }
  ]
}
```

### Format Version 2 Behavior
```sql
select
	data
from
	 pg_logical_slot_peek_changes(
		'test_case', null, null,
		'include-transaction', 'false',
		'format-version', '2'
	) x
```

response
```json
{
  "table": "Test",
  "action": "I",
  "schema": "public",
  "columns": [
    {
      "name": "id",
      "type": "bigint",
      "value": 1
    },
    {
      "name": "mood",
      "type": ""MOOD"", -- invalid json
      "value": "happy"
    }
  ]
}
```

## Expected Behavior
- remove extra quotations to match behavior of table and column escapes
e.g. 
```sql
create type "MOOD" as enum ('happy', 'sad');
```
becomes
```json
...
"columntypes": [
        "bigint",
        "MOOD"
      ],
...
```
instead of `"\"MOOD\""` or `""MOOD""`

Related:
https://github.com/supabase/walrus/issues/31

**Comment by eulerto** [comment]
It works for me. The commit c79a225041cfae566b92177b86635bb48047229f that fixed it was included in wal2json 2.4.

**Comment by olirice** [comment]
thanks for your help
you're right, I was on an old image
confirmed resolved in 2.4
